script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_9_1.txt

```diff
@@ -93,18 +93,18 @@ However.
 It is impossible for a human to cross those two steps.
 <Page26>
 "――――――"
-The room to be able to scream,
-and the ability to feel pain was already frozen in place.
-Was he aware of my plan, or was it just an automatic interception?
-There was a countless number of ice thorns growing around Vlov to greet the attacker.
+My ability to scream,//悲鳴をあげるだけの余裕も、
+and my capacity to feel pain were already frozen solid.//苦痛を感じるだけの感覚も、既に凍っていた。
+Did he notice me, or was it simply an autonomic counterattack?
```

Comment:
I think "automatic interception" is actually more correct and I think we should definitely stick with "interception" over "counterattack", but I guess autonomic can stay since you could read the kanji compound that way even though I don't see it as valid according to any bilingual dictionaries.
> 自動的な迎撃